src/Components/Views/Home/Home.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+
+//importacion de librerias externas
+import { Container, Row, Col } from "react-bootstrap";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import InfiniteScroll from "react-infinite-scroll-component";
+
+//importacion de actions de redux
+
+//importacion de state de redux
+
+//importacion de reducer de redux
+import {
+	getSeriesError,
+	getSeriesPending,
+	getSeries,
+} from "../../../Redux/Reducer/index";
+
+//importacion de componentes
+import CardSerie from "../../Components/CardSerie/CardSerie";
+import {fetchSeries} from "../../../Requests/Requests";
```

Comment:
Add space between the variable. Are you using a linter? This can improve your code readability.